scripts/deploy_tile_env.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```

Comment:
for file name, you meant deploy_til**t**_env.sh most probably?

Reply:
True, fixed.Thanks